cv32/sim/uvmt_cv32/vcs.mk

```diff
@@ -187,12 +187,12 @@ custom: $(VCS_SIM_PREREQ) $(CUSTOM_DIR)/$(CUSTOM_PROG).hex
 
 ################################################################################
 # Explicit target tests
-hello-world:  $(VCS_SIM_PREREQ) $(CUSTOM)/hello_world.hex
+hello-world:  $(VCS_SIM_PREREQ) $(CUSTOM)/hello-world.hex
```

Comment:
I'm conflicted here.  I see what you are doing, but there are a lot of users who are "used to" the underscore.  I've gotten comments about it in the past.   Nevertheless, let's take the hit and clean this up.   I'll approve this.

Reply:
Personally, I would rather have the underscore.  The issue here is that we have to align our agreement on the "name" of the test, or any test, as the yaml file now is a single source that drives the name used throughout the flows and we can't just write a redirection target to rename on-the-fly anymore.  

I thought most folks were used to the hyphen (it's all over our README markdown) so I went that direction.  Would be fine to change it back (and the relevant docs).  Your call.

Reply:
You're right - underscores are all over the READMEs.  Sigh.  `hello-world` it is then.   I'll get a few comments bit when people see the benefits of single-source it will blow over quickly.